tests/ptxn/Driver.toml

```diff
@@ -7,3 +7,12 @@ startDelay = 0
     testName = 'UnitTests'
     maxTestCases = 1
     testsMatching = '/fdbserver/ptxn/test/driver'
+
+	[[test.unittest]]
+	testCaseName = '/fdbserver/ptxn/test/driver'
+	numCommits = 20
+	numTeams = 20
+	numProxies = 1
+	numTLogs = 3
+	numStorageServers = 3
+	messageTransferModel = 0
```

Comment:
Can this be part of `test.workload` specification? If not, can you have multiple `[[test.unittest]]` sections for different `test.workload`?

Reply:
The issue is that in the `test.workload` there is a testsMatching field which allows multiple unit tests in one test suite, so it might be better to differentiate `test.workload` and `test.unittest`. Multiple `test.unittest`s can be used for different `test` sections.